cmd/cluster/kubevirt/create.go

```diff
@@ -76,8 +80,11 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 		// As long as there is no official container image
 		// The image must be provided by user
 		// Otherwise it must fail
-		if opts.KubevirtPlatform.ContainerDiskImage == "" {
-			return errors.New("the container disk image for the Kubevirt machine must be provided by user (\"--containerdisk\" flag)")
+		if (opts.KubevirtPlatform.ContainerDiskImage != "" && opts.KubevirtPlatform.ContainerDiskRepo != "") && (opts.KubevirtPlatform.RootVolumeSize > 0 || opts.KubevirtPlatform.RootVolumeStorageClass != "") {
+			return errors.New("container disk options (\"--containerdisk*\" flag) can not be used together with customized root volume options (\"--root-volume-*\" flags)")
+		}
```

Comment:
I think this is fine for now. We could technically support this combination in the future if we wanted to though.